src/components/shared/ConstructionPage.tsx

```
@@ -0,0 +1,107 @@
+        <div className={`w-20 h-20 md:w-24 md:h-24 mx-auto mb-6 rounded-full flex items-center justify-center ${
+          theme === 'dark' 
+            ? 'bg-orange-500/20 text-orange-400' 
+            : 'bg-orange-100 text-orange-600'
+        }`}>
+          <svg 
+            className="w-10 h-10 md:w-12 md:h-12" 
+            fill="none" 
+            stroke="currentColor" 
+            viewBox="0 0 24 24"
+          >
+            <path 
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)